### ISSUE TITLE
508 Issues - From #2208 (Edit Monitoring Qualifications Data)

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2265]
To be resolved: (Relevant to Edit Monitoring Qualifications Ticket)

- The name of the data table on the qualifications modal under 'Qualification Percent' is generic: aria-label="Data Table". Should be labeled by the qualification percent text.
- When you view qualification percent, the back button is labeled aria-label="go back to systems details". Should be changed to aria-label="go back to Qualification details"

**Comment by dionolympiaCVP** [comment]
Fixed the aria-labels for all of the data tables (Select Configurations, Expanded Configurations Tab, Methods, Systems, Qualifications, etc.)

**Comment by dionolympiaCVP** [comment]
Back buttons aria-labels are fixed for System tables (Fuel Flows/System Components) and Qualification tables (PCT/ LME/LEE)

**Comment by aprematta** [comment]
Looks great, found one minor issue, the aria label is misspelled for Unit Capacities: aria-label="DataTable for Unit Capacitys

**Comment by dionolympiaCVP** [comment]
Fixed typo with aria label for Unit Capacities data table.

![image.png](https://images.zenhubusercontent.com/61436db28447a55f1e8872de/1a708c65-a406-4446-a52a-9eaa14ad3a77)
